.github/workflows/traiage.yaml

```diff
@@ -83,7 +102,8 @@ jobs:
         env:
           CODER_SESSION_TOKEN: ${{ secrets.TRAIAGE_CODER_SESSION_TOKEN }}
           GITHUB_USER_ID: ${{
-            (github.event_name == 'workflow_dispatch' && github.actor_id)
+            (github.event_name == 'workflow_dispatch' && github.actor_id) ||
+            (github.event_name == 'issues' && github.event.issue.user.id)
```

Comment:
What's issue user ID going to be here? Commenter or original author? Latter may be problematic.

Reply:
Ah, I think you're right! I might need to use `github.event.sender` instead.
I wonder if this would just equate to `githbu.actor_id` in the end though?

Reply:
Addressed in https://github.com/coder/coder/pull/20042/commits/e6af33f46d8808307c3253bc508f431757988e60